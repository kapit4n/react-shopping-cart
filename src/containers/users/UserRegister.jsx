import React from 'react'

export default function () {
  return (
    <div>
      Register user
    </div>
  )
}