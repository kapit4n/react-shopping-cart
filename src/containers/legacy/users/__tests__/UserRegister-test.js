import Enzyme, { shallow, mount } from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
Enzyme.configure({ adapter: new Adapter() });
import UserRegister from '../UserRegister'


describe('shallow render Register', () => {
  const wrapper = shallow(<UserRegister />)
  it('snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})

describe('mount render Register', () => {
  const wrapper = mount(<UserRegister />)
  it('snaphost', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
