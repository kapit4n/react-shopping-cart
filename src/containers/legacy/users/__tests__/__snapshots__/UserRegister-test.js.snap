// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mount render App  1`] = `ReactWrapper {}`;

exports[`mount render App snaphost 1`] = `ReactWrapper {}`;

exports[`mount render Register snaphost 1`] = `ReactWrapper {}`;

exports[`shallow render App  1`] = `ShallowWrapper {}`;

exports[`shallow render App snapshot 1`] = `ShallowWrapper {}`;

exports[`shallow render Register snapshot 1`] = `ShallowWrapper {}`;
